index.html

```diff
@@ -125,23 +205,25 @@ <h2 class="project_name">Project name goes here</h2>
                         </ul>
                         <button class="see_project">
                             <p>See this project</p>
-                            <img src="assets/shapes/Union.png" alt="">
+                            <img class="union" src="assets/shapes/Union.png" alt="">
                         </button>
                     </div>
                 </div>
             </div>
         </section>
         <section id="about_myself">
-            <h1>About</h1>
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rerum esse fugiat expedita hic. Quae ullam quis iure nemo nostrum atque. Quisquam omnis aliquid iusto natus nemo non distinctio reiciendis quia!
-            </p>
-            <button>
-                <p>Get my CV</p>
-                <svg width="26" height="25" viewBox="0 0 26 25" fill="none" xmlns="http://www.w3.org/2000/svg">
-                    <path d="M1 6V24H25V6" stroke="white" stroke-width="2"/>
-                    <path d="M13.6623 13.9624L17.9103 9.71441L19.3245 11.1286L12.6622 17.7909L6 11.1286L7.41421 9.71441L11.6623 13.9625V2.52881e-07L13.6623 0V13.9624Z" fill="white"/>
-                    </svg>
-            </button>
+            <div class="about_text">
+                <h1>About</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rerum esse fugiat expedita hic. Quae ullam quis iure nemo nostrum atque. Quisquam omnis aliquid iusto natus nemo non distinctio reiciendis quia!
+                </p>
```

Comment:
- [ ] Dear @Mylo16, Please make sure to use the appropriate fonts for your texts. in this case `'Playfair Display'`. This is to make sure your design looks like the one on the figma design.